src/ctia/flows/crud.clj

```diff
@@ -141,15 +147,20 @@
     (update fm :tempids (fnil into {}) newtempids)))
 
 (s/defn ^:private realize-entities :- FlowMap
-  [{{{:keys [get-in-config]} :ConfigService} :services
+  [{{{:keys [get-in-config]} :ConfigService
+     :as services} :services
     :keys [entities
            flow-type
            identity
            tempids
            prev-entity
            realize-fn] :as fm} :- FlowMap]
   (let [login (auth/login identity)
-        groups (auth/groups identity)]
+        groups (auth/groups identity)
+        realize-fn (MaybeDelayedRealizeFn->RealizeFn
+                     realize-fn
```

Comment:
I'm generally not fan of reusing symbols in a let.
This is generally confusing.
But I see why you did it in this case.

Reply:
I'll remove the surrounding destructuring so it isn't shadowed.